drivers/SmartThings/matter-lock/src/test/test_new_matter_lock.lua

```diff
@@ -755,7 +758,7 @@ test.register_message_test(
       direction = "send",
       message = mock_device:generate_test_message(
         "main",
-        capabilities.lock.lock.locked(
+        capabilities.lock.lock.unlatched(
```

Comment:
I'm not sure we want to change the existing unit test. I suggest adding a new unit test file that sets the `feature_map` to include the `UBOLT` feature and then fully exercising the lock/unlock/unlatch functionality

Reply:
I got it. For that, we need to remove the tests related to unlatch from current test unit. I will move the unlatch test to new test file.